libs/remix-core-plugin/src/lib/compiler-artefacts.ts

```diff
@@ -72,51 +72,61 @@ export class CompilerArtefacts extends Plugin {
     return contractsData
   }
 
-  async getArtefactsFromFE (path, contractName) {
+  _getAllContractArtefactsfromOutput (contractsOutput, contractName) {
```

Comment:
line 31 , probably that we should keep track of the `input` here too, so it can be used throughout the entire app.

Reply:
this is done